app/lib/bk/compat/models/steps/block.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module BK
```

Comment:
while it is not failing, I'd suggest adding the requirements for this file explicitly

```suggestion

require_relative 'base'

module BK
```

Reply:
Adjusted 👍